src/js/common/utils/accessible-modal.js

```diff
@@ -0,0 +1,30 @@
+import { isTab, isReverseTab, getTabbableElements } from './accessibility-helpers';
+
+/*
+ * Creates function that captures Veterans Crisis Line modal focus.
+ */
+
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
Can we change the name of this file to make it more clear that this is for the VCL?

Reply:
Good idea, changed.